File: VB/WindowsFormsApplication25/Form1.vb
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Data
Imports System.Drawing
Imports System.Linq
Imports System.Text
Imports System.Windows.Forms

Namespace WindowsFormsApplication25
    Partial Public Class Form1
        Inherits Form

        Public Sub New()
            InitializeComponent()

        End Sub

        Private Sub checkEdit1_CheckedChanged(ByVal sender As Object, ByVal e As EventArgs) Handles checkEdit1.CheckedChanged
            customBreadCrumbEdit1.Properties.AllowDropDownButtonSelectedState = checkEdit1.Checked
        End Sub
    End Class
End Namespace

File: CS/WindowsFormsApplication25/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Windows.Forms;

namespace WindowsFormsApplication25 {
    public partial class Form1 : Form {
        public Form1() {
            InitializeComponent();
            
        }

        private void checkEdit1_CheckedChanged(object sender, EventArgs e) {
            customBreadCrumbEdit1.Properties.AllowDropDownButtonSelectedState = checkEdit1.Checked;
        }
    }
}
